src/WinRT.Runtime/Marshalers.cs

```diff
@@ -2026,6 +2028,26 @@ static Marshaler()
                 DisposeMarshalerArray = new Action<object>(ABI.System.Type.DisposeMarshalerArray);
                 DisposeAbiArray = new Action<object>(ABI.System.Type.DisposeAbiArray);
             }
+            else if (typeof(Exception).IsAssignableFrom(typeof(T)))

```

Comment:
I wonder if putting this branch after the following (that's checking `IsValueType` could help). Have we seen any regressions at all with these current changes? I can also help validate with sizoscope.

Reply:
As far as I can tell with sizoscope, with the new changes, there doesn't seem to be any unintended additions as there was  with my initial changes.  See https://github.com/microsoft/CsWinRT/pull/1682#discussion_r1694421721